bower.json

```diff
@@ -75,9 +75,6 @@
     "knockout": {
       "js": "dist/knockout.debug.js"
     },
-    "libsodium.js": {
```

Comment:
It's a transitive dependency from `wire-webapp-cryptobox`, so it's in-use! But it's not declared at the top of the `bower.json` file because it comes as a transitive dependency.